MyCryptoHQ/MyCrypto
# Fix Ledger DPaths, Improve Error Messaging

Closes #1515

### Description

Ledger's dpaths were all out of sync with what is in production. This PR fixes that, as well as improves some error messaging.

### Changes

* Fixed all dpaths to match production config
* Added exception to dpath validation for ETC Ledger
* Provided human readable error messages for common Ledger issues
* Added network name to tip since UBQ and EXP have their own Ledger apps (Used to say "ethereum app" all the time.)
* Fixed issue with `.alert` elements having an invisible element that covered other stuff

### Steps to Test

1. Open up this branch and mycrypto.com in side by side tabs
2. Connect to each network on both with a Ledger, confirm addresses are the same
    * If it errors for some reason, confirm it errors the same on both
